src/lumigo_opentelemetry/instrumentations/psycopg/instrumentation.py

```diff
@@ -0,0 +1,262 @@
+# mypy: ignore-errors
```

Comment:
Did you copy this file from elsewhere? why? don't you prefer to import it and maybe patch the specific places that you want to change? 
if only a small portion of this file was change , then it will be easier to maintain

Reply:
i hear you, but i feel like that might be playing with fire - if the dependency's updated, for example

Reply:
we're fixing the dependency's version, so it will not be updated without running the tests, which will catch it if there will be a problem.
That's how we roll in the other instrumentation. IMO this is a cleaner way to do it. your choice

Reply:
it's fair, but i'd prefer to leave the tests separate